blockers.go

```diff
@@ -29,6 +36,37 @@ type ItemInfo struct {
 	Status      string `json:"status"`
 }
 
+func fetchExpressBouncerDownload() (int, error) {
+	now := time.Now()
+	nowFmt := fmt.Sprintf("%d-%d-%d", now.Year(), now.Month(), now.Day())
+	bouncerFirstReleaseDate := "2021-01-01"
+	url := fmt.Sprintf("https://api.npmjs.org/downloads/point/%s:%s/@crowdsec/express-bouncer", bouncerFirstReleaseDate, nowFmt)
+
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return 0, errors.Wrapf(err, "creating request to fetch downloads from NPM API")
+	}
+	client := &http.Client{}
+	resp, err := client.Do(req)
+	if err != nil {
+		return 0, errors.Wrapf(err, "doing request to fetch downloads from NPM API")
+	}
+	if resp.Body != nil {
+		defer resp.Body.Close()
+	}
+
+	body, readErr := ioutil.ReadAll(resp.Body)
```

Comment:
Add a check on `resp.Body`,  if it is `nil` it will segfault (maybe return during the previous check as the data we want is in the body anyway ?)